main.py

```diff
@@ -0,0 +1,114 @@
+import os
+import pygame
+from tkinter import *
+from tkinter.filedialog import askdirectory
+
+root = Tk()
+root.minsize(300, 300)
+root.maxsize(1920, 1080)
+pygame.mixer.init()
+
+songs = []
+current_song = 0
+songs_counter = 0
+
+v = StringVar()
+song_label = Label(root, textvariable=v, width=35)
+
+
+def directory_chooser():
+    global songs_counter
+    directory = askdirectory()
+    os.chdir(directory)
+    
+    for file in os.listdir(directory):
+        if file.endswith(".ogg"):
+            songs.append(file)
+            print(file)
+    
+    pygame.mixer.music.load(songs[current_song])
+    # pygame.mixer.music.play()
+    
+    # PRINTING THE SONG NAMES
+    songs.reverse()
+    songs_counter = len(songs)
+    i = songs_counter
+    for song in songs:
+        listbox.insert(0, str(i) + ". " + song)
+        i -= 1
+    songs.reverse()
+
+
+def change_label():
+    global current_song
+    v.set(songs[current_song])
+
+
+def play_song(event):
+    global current_song
+    pygame.mixer.music.load(songs[current_song])
+    pygame.mixer.music.play()
+    change_label()
+
+
+def stop_song(event):
+    pygame.mixer.music.stop()
+    v.set("")
+
+
+def pause_song(event):
+    pygame.mixer.music.pause()
+
+
+def unpause_song(event):
+    pygame.mixer.music.unpause()
+
+
+def next_song(event):
+    global current_song
+    global songs_counter
+    if current_song < songs_counter - 1:
```

Comment:
Shouldn't i be only song_counter not  song_counter - 1, because is we are on the song before the last one for example how are we going to skip it?

Reply:
The songs_counter variable is a number counted from 1 up. The current_song variable is a number starting from 0 because it is used to iterate through an array. So, if it's "current_song < songs_counter" we will be able to get out of the array's reach, which will break the program.